### ISSUE TITLE
 Change position and wording of help text for the number spinners (State Descriptions)

### ISSUE THREAD

**Issue by terracoda** [phetsims/ratio-and-proportion#135]
Please change position and wording of help text for the number spinners.

Current help text which currently comes after the number spinners is:
- Set left hand and right hand values to explore custom challenge

Please move help text so it comes before the two number spinners.

Here's the new help text:
- Set left hand and right hand values to create your custom ratio challenge.

**Comment by zepumph** [comment]
It is easy to make the help text go below the number spinners, or above the "my challenge" button. It is more challenging to place it in between the two. It is still possible, but will involve changes in AccordionBox to make it possible.

If you would like this to be done, will you please discuss if you feel like this is a general feature of all accordion boxes, or just where we want the help text for this individual instance?

**Comment by terracoda** [comment]
@zepumph, would the help text be outside the accordion box if it came before the number spinners? I was imagining html like this:

- button+heading for title of box
  - open wrapper for number spinners 
    - p for help text
    - left hand number spinner
    - right hand number spinner
  - close wrapper for box

Obviously, that's very simplified.

It's not a show stopper to leave the help text where it is. We can see how it works out for learners in the next round of interviews. I don't think there were any specific comments from the interviewees. I just requested the change in order because it made more sense to me.

**Comment by zepumph** [comment]
@terracoda mentioned that this help text is not for the accordion box, but instead for the content of the accordion box.  We want the help text above that content. That should be easy! One implementation coming right up!

**Comment by zepumph** [comment]
Fixed in master, please review.

**Comment by terracoda** [comment]
You have the help text where I wanted it now. Nice work, @zepumph and thanks. Closing this issue.
